dbt/adapters/snowflake/connections.py

```diff
@@ -59,8 +59,8 @@ class SnowflakeCredentials(Credentials):
     proxy_host: Optional[str] = None
     proxy_port: Optional[int] = None
     protocol: Optional[str] = None
-    connect_retries: int = 0
-    connect_timeout: int = 10
+    connect_retries: int = 1
```

Comment:
👍 

Also discussed: using `connect_retries` as the standard name everywhere (including Postgres/Redshift)